cmd/bzzd/main.go

```diff
@@ -85,6 +85,10 @@ var (
 		Usage: "URL of the Ethereum API provider",
 		Value: node.DefaultIPCEndpoint("geth"),
 	}
+	CorsString = cli.StringFlag{
```

Comment:
Please append `Flag` to the variable name. Bonus points for fixing up the others, too :-D